src/modules/hide_sidebar_elements/index.js

```diff
@@ -80,6 +82,25 @@ class HideSidebarElementsModule {
     }
   }
 
+  loadHideOpenStories() {
+    const setting = settings.get(SettingIds.SIDEBAR);
+    const enabled = hasFlag(setting, SidebarFlags.OPEN_STORIES);
+    const [element] = document.getElementsByClassName(styles.hideOpenStories);
+
+    if (enabled && element == null) {
+      const sidenav = document.querySelector('.side-bar-contents');
+      const node = twitch.getOpenStoriesDOMNode(sidenav);
```

Comment:
do we need to have this be under a dom watcher like the other sidebar elements?

Reply:
You're right, we do